web/templates/personal_menu.hbs

```diff
@@ -14,15 +14,13 @@
                 <p class="user-type">{{user_type}}</p>
             </div>
         </header>
-        <section class="popover-menu personal-menu-actions" data-user-id="{{user_id}}">
-            <ul class="list">
+        <section class="dropdown-menu-list-section personal-menu-actions" data-user-id="{{user_id}}">
+            <ul>
                 <li class="text-item hidden-for-spectators">
                     <i class="zulip-icon zulip-icon-clock"></i>
                     {{#tr}}{user_time} local time{{/tr}}
                 </li>
-            </ul>
-
-            <ul class="list">
+                <div class="navbar-dropdown-divider"></div>
```

Comment:
Again, I don't see why the dividers can't be handled as simple borders on these grouping `<ul>` elements.